src/lib/utils/__tests__/createSendEventForFacet-test.ts

```diff
@@ -0,0 +1,104 @@
+import algoliasearchHelper from 'algoliasearch-helper';
+import { createSendEventForFacet } from '../createSendEventForFacet';
+import { InstantSearch, SearchClient } from '../../../types';
+
+jest.mock('../isFacetRefined', () => jest.fn());
+
+import isFacetRefined from '../isFacetRefined';
+
+const instantSearchInstance = {} as InstantSearch;
+let sendEvent;
+let helper;
+beforeEach(() => {
```

Comment:
would like to see this as a constructor instead of beforeEach too (easier to see where variables are from, and avoids mutation / global variables everywhere)

Reply:
thanks for the reminder: https://github.com/algolia/instantsearch.js/pull/4464/commits/669501559bff1fda2a340c02658b87fe574053a9